src/components/sections/contact-banner.json

```diff
@@ -14,6 +15,11 @@
       "type": "relation",
       "relation": "oneToOne",
       "target": "api::theme.theme"
+    },
+    "socialMedias": {
+      "type": "relation",
+      "relation": "oneToMany",
+      "target": "api::social-media.social-media"
```

Comment:
Request:
Could you please, fix the SocialMedia type? Currently, it looks like 
<img width="277" alt="image" src="https://github.com/user-attachments/assets/b7e60d15-22af-4c8c-a539-c20ca41ebd54">

But it should contain a link to the image. And, probably, the theme is redundant here.
Also, it's unclear what title, text, and type are responsible for.